mobx/lib/mobx.dart

```diff
@@ -30,7 +30,7 @@ library mobx;
 
 export 'package:mobx/src/api/action.dart';
 export 'package:mobx/src/api/annotations.dart'
-    show action, computed, observable, toString;
+    show action, computed, observable, MakeStoreConfig;
```

Comment:
The annotation should be called `storeConfig` and exported as such. `MakeStoreConfig` is only the type of the annotation

Reply:
You mean StoreConfig to deal with parameter parts ?

Reply:
> You mean StoreConfig to deal with parameter parts ?

Yes, let's call it `StoreConfig`